homework_36/src/main/java/org/spring_framework/ProductRepository.java

```diff
@@ -0,0 +1,31 @@
+package org.spring_framework;
+
+import org.spring_framework.entity.Product;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+public class ProductRepository {
+    private List<Product> products;
+
+    public ProductRepository() {
+        this.products = new ArrayList<>();
+        products.add(new Product(192, "apple", 12.3));
```

Comment:
You can add some values to products by using @PostConstruct or its analogue annotation instead of doing it via constructor.